From d8d20f1a7b889ee28f786cc1dfaa65b610d63b13 Mon Sep 17 00:00:00 2001
From: Franklin S. Cooper Jr <fcooper@ti.com>
Date: Wed, 31 Oct 2012 21:06:01 -0500
Subject: [PATCH] make demos maximized for TISDK use

* Maximize the weatherinfo and imageviewer demos for use in the TISDK

Signed-off-by: Franklin S. Cooper Jr <fcooper@ti.com>
---
 demos/embedded/weatherinfo/weatherinfo.cpp |    5 -----
 examples/widgets/imageviewer/main.cpp      |    4 ----
 2 files changed, 0 insertions(+), 9 deletions(-)

diff --git a/demos/embedded/weatherinfo/weatherinfo.cpp b/demos/embedded/weatherinfo/weatherinfo.cpp
index 0614149..c47504d 100644
--- a/demos/embedded/weatherinfo/weatherinfo.cpp
+++ b/demos/embedded/weatherinfo/weatherinfo.cpp
@@ -492,12 +492,7 @@ int main(int argc, char *argv[])
     QApplication app(argc, argv);
 
     WeatherInfo w;
-#if defined(Q_OS_SYMBIAN)
     w.showMaximized();
-#else
-    w.resize(520, 288);
-    w.show();
-#endif
 
     return app.exec();
 }
diff --git a/examples/widgets/imageviewer/main.cpp b/examples/widgets/imageviewer/main.cpp
index 8d1a068..35fb80d 100644
--- a/examples/widgets/imageviewer/main.cpp
+++ b/examples/widgets/imageviewer/main.cpp
@@ -46,10 +46,6 @@ int main(int argc, char *argv[])
 {
     QApplication app(argc, argv);
     ImageViewer imageViewer;
-#if defined(Q_OS_SYMBIAN)
     imageViewer.showMaximized();
-#else
-    imageViewer.show();
-#endif
     return app.exec();
 }
-- 
1.7.0.4

